src/pages/documentation/edge-network-extensions/api-reference.md

```diff
@@ -50,6 +50,37 @@ Resets current state of the AEP Edge extension and clears previously cached cont
 
 See [MobileCore.resetIdentities](../mobile-core/api-reference.md#resetidentities) for more details.
 
+## sendEvent
+
+Sends an Experience event to Adobe Experience Platform Edge Network.
+
+<TabsBlock orientation="horizontal" slots="heading, content" repeat="2"/>
+
+Android
+
+<Tabs query="platform=android&api=send-event"/>
+
+iOS (AEP 3.x)
+
+<Tabs query="platform=ios-aep&api=send-event"/>
+
+## setLocationHint
+
+Sets the Edge Network location hint used in requests to the Adobe Experience Platform Edge Network. Passing nil or an empty string clears the existing location hint. Edge Network responses may overwrite the location hint to a new value when necessary to manage network traffic.
+
+<InlineAlert variant="warning" slots="text"/>
+Use caution when setting the location hint. Only use location hints for the "EdgeNetwork" scope. An incorrect location hint value will cause all Edge Network requests to fail with 404 response code.
```

Comment:
Just a small thing, but I think the renderer is a bit finnicky and it needs the newline between the <InlineAlert> and the text!

Reply:
I tested locally and it looked fine. I added the newline, though. If nothing else it makes the raw file easier to read with the newline.